### PR TITLE
Release 1.3.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Review by blackbaud-johnly — DISMISSED** [review]
I tweaked the wording of the second item. LGTM.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 1.3.1 (2019-10-07)
+
+- Fixed the list data provider demo to show all data provider functionality. [#23](https://github.com/blackbaud/skyux2-demos/pull/23)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: Should this be more specific? Something like...
```
...show examples of sorting, filtering, and searching while using the data provider.
```

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: I went back and forth. I'm fine with being more specific @Blackbaud-AlexKingman. Updated

@blackbaud-johnly care to give it another review with the new wording?

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Review by blackbaud-johnly — APPROVED** [review]
LGTM

**Pull request merged**